CHANGELOG.md

```diff
@@ -1,5 +1,78 @@
+ * Report only XML syntax error for *.exsd files. See [#782](https://github.com/redhat-developer/vscode-xml/pull/782). 
+ * Place relevant information on the first line of error messages. See [lemminx#1145](https://github.com/eclipse/lemminx/issues/1145).
+ * Validate uri of XML catalog. See [lemminx#823](https://github.com/eclipse/lemminx/issues/823).
+ * Inadequate error message for download error. See [#763](https://github.com/redhat-developer/vscode-xml/issues/763). 
+ * CodeAction/Completion to insert required attributes from XSD. See [lemminx#365](https://github.com/eclipse/lemminx/issues/365).
+ * CodeAction for `cvc-complex-type.2.4.b`: `insert all expected` vs. `insert all possible`. See [lemminx#1255](https://github.com/eclipse/lemminx/issues/1255).
+ * Do not complete paths in attr unless beginning of value looks like a path. See [lemminx#1293](https://github.com/eclipse/lemminx/pull/1293).
+ * Add command `Restart Language Server`. See [#539](https://github.com/redhat-developer/vscode-xml/issues/539). 
+ * Use the HTTP proxy configuration for HTTPS as well. See [lemminx#1253](https://github.com/eclipse/lemminx/pull/1253).
+ * Add usesOnlineServices tag. See [#701](https://github.com/redhat-developer/vscode-xml/pull/701).
+
+### Bug Fixes
+
+ * Autocompletion of attribute values won't display documentation. See [#736](https://github.com/redhat-developer/vscode-xml/issues/736).
+ * Null pointer exception when processing code action `cvc_complex_type_2_4_b`. See [lemminx#1280](https://github.com/eclipse/lemminx/issues/1280).
+ * Annotations found in base complex type not shown on hover. See [lemminx#1268](https://github.com/eclipse/lemminx/pull/1268).
+ * Format is wrong with multiple line content. See [#600](https://github.com/redhat-developer/vscode-xml/issues/600).
+ * Format CDATA sections without adding new lines. See [lemminx#1193](https://github.com/eclipse/lemminx/issues/1193).
+ * Formatting with `xml.format.emptyElements`: `expand` chokes on malformed XML. See [lemminx#650](https://github.com/eclipse/lemminx/issues/650).
+ * Fix format problem with multiple root elements. See [#634](https://github.com/redhat-developer/vscode-xml/issues/634).
+ * Remove darwin-arm64 platform from generic publishing call.. See [#734](https://github.com/redhat-developer/vscode-xml/pull/734).
+ * Server binaries for "osx-x86_64" are not found. See [#729](https://github.com/redhat-developer/vscode-xml/issues/729).
+
+### Build
+
+ * Make sure that 'vsce' is installed in nodes that use it.. See [#789](https://github.com/redhat-developer/vscode-xml/pull/789).
+ * Fix problem matcher. See [#779](https://github.com/redhat-developer/vscode-xml/pull/779).
+ * Explicitly set 'main' as target branch for cloning of lemminx repo.. See [#750](https://github.com/redhat-developer/vscode-xml/pull/750).
+ * Update Jenkins build scripts for branch renaming. See [#747](https://github.com/redhat-developer/vscode-xml/pull/747).
```

Comment:
This entry can probably just be removed. Either that or include it as part of the "Explicitly set 'main' ...".

It was mainly a fix that was missed when we changed branch names.

Reply:
removed